web/src/OverviewTable.test.tsx

```diff
@@ -55,6 +60,31 @@ const tableViewWithSettings = ({
   )
 }
 
+const findTableColumnName = (
+  wrapper: ReactWrapper<any>,
+  columnName: string,
+  sortable = true
+) => {
+  const selector = sortable ? `Sort by ${columnName}` : columnName
+  return wrapper.find(ResourceTableHeader).filter(`[title="${selector}"]`)
+}
+
+const findTableColumn = (wrapper: ReactWrapper<any>, columnName: string) => {
```

Comment:
gosh, yeah. so:
- first helper returns the table header. the table header can be clicked to trigger sorts, and it's also inspected to test any conditional display logic that might occur within the table header.
- second helper returns the column data model, so that the column properties can be tested. i opted to test the column properties (like `isSorted`) directly as the source of truth, rather than rely on our implementation of the `.isSorted` class. that may be splitting hairs, but i figured it was a better test approach.

naming these in a clear way is tough, partially because of how we map through and component-ize the different table elements. i think the tests (and code itself) would be clearer if we broke the nested `map`s in the jsx into individual components, but that refactoring seemed beyond the scope of this pr. i made some naming changes that hopefully improve clarity!

Reply:
Ah! My issue was that when I saw "column", I thought of its content rather than its metadata, but react-table regularly uses "column" to refer to the metadata, so I just need to learn.

Throwing an explicit return type of like `: ColumnInstance<RowValues>[]` on there might help? nbd though

Reply:
done!